### PR TITLE
chore: use golang build tools image

### LINKED ISSUE #9: Use golang build tools image

### PR DESCRIPTION
**Pull request by jhedev**
Closes #9

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,5 +1,10 @@
-GO_VERSION=1.9
-GO_FILES=$(shell find . -type f -name "*.go")
+GO_FILES := $(shell find . -type f -name "*.go")
+GO_BUILD := CGO_ENABLED=0 go build -ldflags="-w -s"
+GO_TOOLS := gridx/golang-tools:master-839443d
```
Comment (gq0) [inline Makefile:3]: This is a certain go version then, right? Is it possible to highlight the used go version?

File: Makefile [hunk 2]
```diff
@@ -1,5 +1,10 @@
-GO_VERSION=1.9
-GO_FILES=$(shell find . -type f -name "*.go")
+GO_FILES := $(shell find . -type f -name "*.go")
+GO_BUILD := CGO_ENABLED=0 go build -ldflags="-w -s"
```
Comment (gq0) [inline Makefile:2]: This is not used anymore, or?

File: Makefile (hunk 1 above)
Comment (jhedev) [reply Makefile:3]: This is not only a certain go version but also a certain version of glide and golint. Yeah, I guess I should somehow reflect it in the tag. I'll open an issue in the grid-x/images repo for this (where the dockerfile is located).

File: Makefile (hunk 2 above)
Comment (jhedev) [reply Makefile:2]: Oh good catch. It should be in the build targets. I'll fix it.

Comment (jhedev) [reply Makefile:2]: pushed a fixup

**Review by gq0 — APPROVED** [review]

**Review by philippfranke — CHANGES_REQUESTED** [review]
Branch is outdated

**Comment by jhedev** [comment]
@philippfranke fixed

**Review by philippfranke — APPROVED** [review]

**Pull request merged**